### PR TITLE
delay task for server phase

### PR DESCRIPTION
**Pull request by sureshtadisetty**

### CODE REVIEW

File: Tasks/Delay/task.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "id": "28782B92-5E8E-4458-9751-A71CD1492BAE",
+    "name": "Delay",
+    "friendlyName": "Delay",
+    "description": "Pause and wait for specified number of minutes.",
+    "helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=825235)",
+    "category": "Utility",
+    "visibility": [
+        "Release"
```
Comment (bansalaseem) [inline Tasks/Delay/task.json]: Add Build

File: Tasks/Delay/task.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+    "id": "28782B92-5E8E-4458-9751-A71CD1492BAE",
+    "name": "Delay",
```
Comment (bansalaseem) [inline Tasks/Delay/task.json]: Add Preview flag and make sure that it does not go in onprem tfs

Comment (sureshtadisetty) [inline Tasks/Delay/task.json]: As discussed, not adding Preview flag as task is not being ported to M122 snapshot.

File: Tasks/Delay/task.json (hunk 1 above)
Comment (sureshtadisetty) [inline Tasks/Delay/task.json]: Done.

**Pull request merged**